jobs/infra/playbook-jenkins.yml

```diff
@@ -164,11 +163,22 @@
       command: "snap remove {{item}}"
       ignore_errors: yes
       loop:
+        - "aws-cli"
         - "microk8s"
         - "bundletester"
         - "surl"
       tags:
         - jenkins
+    - name: install awscli
+      shell: |
+        curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
```

Comment:
this will run on all jenkins workers (amd, arm, s390x).  if you *really* only want amd64 installed, you should wrap this shell command in a "if arch == amd64".

also consider the [more recent amz-aws-cli snap](https://snapcraft.io/amz-aws-cli) if you don't want to go through this slightly more complicated shell script.

Reply:
I have questions regarding this. I would much prefer using the more recent snap, but maintenance is an unknown. And I believe this snap is also just for `amd64`. 

We would need these calls in arm as well, could we make a snap and maintain it ourselves? But I believe there are only amd64 and arm installers from aws for the cli v2. Maybe using the `awscli` [apt package](https://packages.ubuntu.com/jammy/awscli) is better, since it supports all architectures. Only downside we will be following an older version (1.22 something currently).

@ktsakalozos I would like to hear your thoughts on this as well. I think the apt package would work for us.

Reply:
@berkayoz lets go with the apt package. Getting the snap will limit us to amd64 and always getting the latest from the .zip may introduce some flakiness.